make_profiler/lint_makefile.py

```
@@ -123,22 +133,43 @@ def validate_spaces(lines: List[str], *, errors: List[str] | None = None) -> boo
                 msg = f"Trailing spaces ({i}): {line}"
                 print(msg)
                 if errors is not None:
-                    errors.append(msg)
+                    errors.append(
+                        LintError(
+                            type="trailing spaces",
+                            message=msg,
+                            line_number=i,
+                            line_text=line,
+                        )
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)